Packages/com.unity.inputsystem/InputSystem/Editor/DeviceSimulator/InputSytemPlugin.cs

```diff
@@ -0,0 +1,92 @@
+#if UNITY_EDITOR && UNITY_2021_1_OR_NEWER
+
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.DeviceSimulation;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+
+public class InputSystemPlugin : DeviceSimulatorPlugin
```

Comment:
Should be in namespace.

Also, can this be internal? If it has to be public, needs to be documented.

Reply:
Yup, it can be internal, fixed it.